lib/attack_test.go

```diff
@@ -358,3 +359,19 @@ func TestClient(t *testing.T) {
 		t.Errorf("Expected timeout error")
 	}
 }
+
+func TestSeqHeader(t *testing.T) {
+	t.Parallel()
+
+	var got http.Header
+
+	atk := NewAttacker()
+	res := atk.hit(func(trt *Target) error {
+		got = trt.Header
+		return nil
+	}, "")
+
+	if got.Get("X-Vegeta-Seq") != strconv.FormatUint(res.Seq, 10) {
+		t.Fatalf("got: %v, want: %v", got.Get("X-Vegeta-Seq"), res.Seq)
+	}
```

Comment:
I'm wondering if it would be better to have a small loop here to make sure that we are not only getting the zero value. Here `res.Seq` is equal to 0.